ch32v003fun/ch32v003fun.h

```diff
@@ -13794,7 +13794,29 @@ void SystemInit(void);
 // Put an output debug UART on Pin D5.
 // You can write to this with printf(...) or puts(...)
 
-#include <setjmp.h>
+#ifdef __riscv
```

Comment:
Is there any point in this check since `ch32v003fun.h` is implicitly for RISC-V?

Reply:
Just habit, I can remove it if you want.